connector_importer_source_odoorpc/components/importer.py

```diff
@@ -29,6 +29,12 @@ def _record_lines(self):
                 followed_mapping[line['_model']][line['_line_nr']] = line
         # set it on shared store
         self.recordset.set_shared({'followed_mapping': followed_mapping})
+        # Without invalidating cache we will have a bug because of Serialized
+        # field in odoo. It uses json.loads on convert_to_cache, which leads
+        # to all of our int dict keys converted to strings. Except for the
+        # first value get, where we get not from cache yet. So in order to
+        # streamline this I invalidate cache right away and use string keys
+        self.recordset.invalidate_cache(('shared_data',))
```

Comment:
as `set_shared` is an API method from us, put invalidate cache inside it (w/ the comment ;))